--- crash-7.3.2/memory.c.orig
+++ crash-7.3.2/memory.c
@@ -19412,9 +19412,8 @@ freelist_ptr(struct meminfo *si, ulong ptr, ulong ptr_addr)
 	if (VALID_MEMBER(kmem_cache_random)) {
 		/* CONFIG_SLAB_FREELIST_HARDENED */
 
-		if (THIS_KERNEL_VERSION >= LINUX(5,7,0))
-			ptr_addr = (sizeof(long) == 8) ? bswap_64(ptr_addr)
-						       : bswap_32(ptr_addr);
+		ptr_addr = (sizeof(long) == 8) ? bswap_64(ptr_addr)
+					       : bswap_32(ptr_addr);
 		return (ptr ^ si->random ^ ptr_addr);
 	} else
 		return ptr;
-- 
fix freepointer issue
